server/paymentProviders/stripe/webhook.ts

```diff
@@ -160,7 +160,7 @@ export const paymentIntentSucceeded = async (event: Stripe.Event) => {
   const paymentIntent = event.data.object as Stripe.PaymentIntent;
   const order = await models.Order.findOne({
     where: {
-      status: OrderStatuses.PROCESSING,
+      status: [OrderStatuses.NEW, OrderStatuses.PROCESSING],
```

Comment:
This code is currently changing in https://github.com/opencollective/opencollective-api/pull/8497

Reply:
Looks like that was merged, so this needs to be rebased/adjusted?